EpiserverRedirects/Encoder/UrlPathSpaceEncoder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Forte.EpiserverRedirects.Model;
+
+namespace Forte.EpiserverRedirects.Encoder
+{
+    public class UrlPathSpaceEncoder : IUrlPathEncoder
+    {
+        public UrlPath Encode(UrlPath urlPath)
+        {
+            var path = urlPath.ToString();
+            var encodedUri = Uri.EscapeUriString(path);
```

Comment:
`UrlPath` has method `NormalizePath` - maybe we could make this logic (escapeUriString) a part of that method? It would be much more concise

Reply:
I moved that logic into new `UrlPath` method named `FromUriPathEncode`